overlord/snapshotstate/backend/backend.go

```diff
@@ -323,3 +327,150 @@ func addDirToZip(ctx context.Context, snapshot *client.Snapshot, w *zip.Writer,
 
 	return nil
 }
+
+type exportMetadata struct {
+	Format int       `json:"format"`
+	Date   time.Time `json:"date"`
+	Files  []string  `json:"files"`
+}
+
+// Export allows exporting a given snapshot set
+func Export(ctx context.Context, setID uint64) (*SnapshotExport, error) {
```

Comment:
A bit unclear why to keep this and not just the New... ?

Reply:
Dropped now, thanks